antenna-workflow-steps/analyzers/antenna-ort-result-analyzer/src/test/resources/scan-result.yml

```diff
@@ -0,0 +1,137 @@
+---
```

Comment:
why is there the `---` ?

Reply:
This is part of the result file given out by ORT, I have left it in to keep it as close to a real result as possible and to be sure that it would not lead to a parsing error with the yaml-Mapper